src/main/java/ch/njol/skript/effects/EffBroadcast.java

```diff
@@ -42,49 +42,69 @@
 		"message to all players instead of broadcasting it."})
 @Examples({"broadcast \"Welcome %player% to the server!\"",
 		"broadcast \"Woah! It's a message!\""})
-@Since("1.0")
+@Since("1.0, INSERT VERSION (send multiple times)")
 public class EffBroadcast extends Effect {
+
 	static {
-		Skript.registerEffect(EffBroadcast.class, "broadcast %strings% [(to|in) %-worlds%]");
+		Skript.registerEffect(EffBroadcast.class, "broadcast %strings% [(to|in) %-worlds%] [%number% times]");

```

Comment:
you gotta mark `%number%` as nullable otherwise this will cause an exception the repeat isn't not used